File: c/Array/findUnique.c
#include<stdio.h>
#include<stdbool.h>
int main(){
    int m,a;
     printf("enter no of elements");
    scanf("%d",&m);
    int arr[m];
   printf("enter all the elements");
    for(int i=0;i<m;i++)
        scanf("%d",&arr[i]);
        for (int i=0;i<m;i++){
            bool flag=true;
            for (int j=i+1;j<m;j++){
                if(arr[i]==arr[j])
                 flag=false;
                  if (flag==true)  { 
                    printf("%d is unique",arr[i]);
                 break;}
                 }
            }
        
        
        return 0;}

File: java/NumberSystems/FindUnique.java
package NumberSystems;

public class FindUnique {
      public static void main(String[] args) {
        int[] arr = {1,2,2,1,3,5,3};
        System.out.println(ans(arr));
      }   
      private static int ans(int[] arr){
        int unique = 0;
        for(int n : arr){
            unique = unique^n;
        }   return unique;
      }
}
